gitops/azure-devops/BuildAppImage_ADOReleases.md

```diff
@@ -112,4 +134,39 @@ YAML_PATH_VALUE: the value to the subkey
 
 After the Release runs successfully, the new application image that was generated in the Pipeline Build (Step #2) should now be referenced appropriately in the HLD.
 
-## [High Level Definition (HLD) -> K8s Manifests](https://github.com/Microsoft/bedrock/blob/master/gitops/azure-devops/README.md)
+### 5. Update Manifest Generation Pipeline To Be Environment Aware
+
+Now that we have created a release pipeline with environment specific configurations we need to make sure that manifest generation pipeline knows to generate yaml for these environments. Below is a snippet from an example [azure-pipeline.yaml](https://github.com/andrebriggs/fabrikate-go-server/blob/master/azure-pipelines.yml) build file in a high-level definition repo. 
+
+<pre>
+ - task: ShellScript@2
+    displayName: Transform fabrikate definitions and publish to YAML manifests to repo
+    inputs:
+      scriptPath: build.sh
+    condition: 'ne(variables[''Build.Reason''], ''PullRequest'')'
+    env:
+      ACCESS_TOKEN_SECRET: $(ACCESS_TOKEN)
+      COMMIT_MESSAGE: $(Build.SourceVersionMessage)
+      REPO: $(MANIFEST_REPO)
+      <b>FAB_ENVS: 'DEV,QA,STAGING WEST,STAGING EAST,STAGING CENTRAL,PROD WEST,PROD EAST,PROD CENTRAL'</b>
+</pre>
```

Comment:
Maybe we should add `BRANCH_NAME: $(Build.SourceBranchName)` as an example too, in case any other branch name is being pushed to hld?